google/cloud/forseti/common/util/file_uploader.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility functions for file uploads."""
+
+
+import tempfile
+
+from google.cloud.forseti.common.data_access import csv_writer
+from google.cloud.forseti.common.gcp_api.storage import StorageClient
+from google.cloud.forseti.common.util import logger
+from google.cloud.forseti.common.util import parser
+
+
+LOGGER = logger.get_logger(__name__)
+
+
+def upload_json(data, gcs_upload_path):
+    """Upload inventory summary/violation data in json format.
```

Comment:
Remove 'inventory summary/violation` from doc here and method below.

Reply:
Done. Re. "how are we setting `tmp_data.name` to be `inv_summary`": It is the `gcs_upload_path` [set here](https://github.com/GoogleCloudPlatform/forseti-security/blob/71f7ff261529e35af34e4149cc366dc31664db57/google/cloud/forseti/notifier/notifiers/inventory_summary.py#L70).
I will expand `inv_summary` to `inventory_summary` as requested. Re. the 2 timestamps:
 * the first time stamp is the `inventory_index_id`
 * the second time stamp is the date/time at which the notification was emitted

I am fine with removing either of them. Please let me know.